src/transformers/tokenization_bart.py

```diff
@@ -45,6 +45,79 @@ class BartTokenizer(RobertaTokenizer):
         "merges_file": {m: merges_url for m in _all_bart_models},
     }
 
+    def prepare_seq2seq_batch(
+        self,
+        src_texts: List[str],
+        tgt_texts: Optional[List[str]] = None,
+        max_length: Optional[int] = None,
+        max_target_length: Optional[int] = None,
+        padding: str = "longest",
+        return_tensors: str = "None",
+        **kwargs,
+    ) -> BatchEncoding:
+        r"""
+        Prepare a batch that can be passed directly to an instance of :class:`~transformers.BartModel`.
+
+        Args:
+            src_texts (:obj:`List[str]`):
+                List of input texts.
+            tgt_texts (:obj:`List[str]`, `optional`):
+                List of target texts.
+            max_length (:obj:`int`, `optional`):
+                Maximum length for the source texts. If not provided, this will use the predefined model maximum length.
+            max_target_length (:obj:`int`, `optional`):
+                Maximum length for the target texts. If not provided, this will use the predefined model maximum length.
+            padding (:obj:`bool`, :obj:`str` or :class:`~transformers.tokenization_utils_base.PaddingStrategy`, `optional`, defaults to :obj:`False`):
+                Select a strategy to pad the returned sequences (according to the model's padding side and padding
+                index) among:
+
+                * :obj:`True` or :obj:`'longest'`: Pad to the longest sequence in the batch (or no padding if only a
+                  single sequence if provided).
+                * :obj:`'max_length'`: Pad to a maximum length specified with the argument :obj:`max_length` or to the
+                  maximum acceptable input length for the model if that argument is not provided.
+                * :obj:`False` or :obj:`'do_not_pad'` (default): No padding (i.e., can output a batch with sequences of
+                  different lengths).
+            return_tensors (:obj:`str` or :class:`~transformers.tokenization_utils_base.TensorType`, `optional`):
+                If set, will return tensors instead of list of python integers. Acceptable values are:
+
+                * :obj:`'tf'`: Return TensorFlow :obj:`tf.constant` objects.
+                * :obj:`'pt'`: Return PyTorch :obj:`torch.Tensor` objects.
+                * :obj:`'np'`: Return Numpy :obj:`np.ndarray` objects.
+            **kwargs:
+                Additional keyword arguments passed along to :obj:`self.__call__`.
+        Returns:
```

Comment:
There is a new docstring on master/ tokenization_utils_base.py that you may want to (a) reuse or (b) modify.